src/app/omsorgspenger-utbetaling/components/VarigEndring.tsx

```diff
@@ -0,0 +1,54 @@
+import DatoInputFormik from 'app/components/formikInput/DatoInputFormik';
+import RadioPanelGruppeFormik from 'app/components/formikInput/RadioPanelGruppeFormik';
+import TextAreaFormik from 'app/components/formikInput/TextAreaFormik';
+import TextFieldFormik from 'app/components/formikInput/TextFieldFormik';
+import VerticalSpacer from 'app/components/VerticalSpacer';
+import { JaNei } from 'app/models/enums';
+import intlHelper from 'app/utils/intlUtils';
+import { kunTall } from 'app/utils/patterns';
+import { Field, FieldProps } from 'formik';
+import { capitalize } from 'lodash';
+import React from 'react';
+import { useIntl } from 'react-intl';
+
+export default function VarigEndring() {
+    const intl = useIntl();
+    return (
+        <Field name="opptjeningAktivitet.selvstendigNaeringsdrivende.info.erVarigEndring">
+            {({ field, form }: FieldProps<boolean>) => (
+                <>
+                    <RadioPanelGruppeFormik
+                        legend={intlHelper(intl, 'skjema.sn.varigendring')}
+                        checked={field.value ? 'ja' : 'nei'}
```

Comment:
ikke true eller false for checked?

Reply:
Må være string i radio-grupper, mens punsj-backend vil ha boolean. Mulig det heller skulle blitt mappet om ved mellomlagring og innsending